tests/test_helpers.py

```diff
@@ -560,6 +561,24 @@ def test_attachment(self):
             assert options['filename'] == 'index.txt'
             rv.close()
 
+    def test_attachment_with_utf8_filename(self):
+        app = flask.Flask(__name__)
+
+        with app.test_request_context():
+            rv = flask.send_file(
+                'static/index.html', as_attachment=True,
+                attachment_filename=u'Ñandú／pingüino.txt'
+            )
+            value, options = parse_options_header(
+                rv.headers['Content-Disposition']
+            )
+            rv.close()
+
+        assert value == 'attachment'
+        assert sorted(options.keys()) == ('filename', 'filename*')
+        assert options['filename'] == 'Nandu/pinguino.txt'
+        assert options['filename*'] == 'UTF-8''%C3%91and%C3%BA%EF%BC%8Fping%C3%BCino.txt'
```

Comment:
That's my bad, the outer quotes should be `"`. Then again, this part of the test already fails because the option key is 'filename' at this point.